content/posts/newsletter/2018-05.md

```diff
@@ -0,0 +1,108 @@
+It was never used at scale, and was mainly relegated to chat bots that needed
+compatibility with Google Talk. However, as Google phased out XMPP federation,
+such usefulness dwindled.
+
+### Events
+
+An XMPP sprint is being organized for August in Cambridge, UK.
+The event will take place in the Collabora offices and dates are still to be
+finalized. You still have a chance to vote for your preferred date and to suggest
+topics for the sprint. Visit [https://col.la/xmppsprint](https://col.la/xmppsprint)
+and join the groupchat [xmpp-sprint@chat.cluxia.eu](xmpp:xmpp-sprint@chat.cluxia.eu?join).
+
+### GDPR
+
+On 25th May the European Union's "General Data Protection Regulation" has come
+into force. The XMPP community has been discussing its impact and have come up
+with various ideas and strategies on how to ensure compliance.
+
+Discussions happened regularly in the [xsf@muc.xmpp.org](xmpp:xsf@muc.xmpp.org) groupchat.
+
+There is a [GDPR page on the XMPP wiki](https://wiki.xmpp.org/web/GDPR) which
+collects and summarises the results of these discussions.
+
+Winfried Tilanus, who's been active in these discussions, created a new
+proto-XEP: [Best practices for GDPR compliant deployment of XMPP](https://xmpp.org/extensions/inbox/gdpr.html)
+
+
+## Software releases
+
+### Servers
+
+* [Prosody 0.9.13](http://blog.prosody.im/prosody-0-9-13-released/?utm_source=xmppnewsletter):
+  A new maintenance release to the old stable 0.9 branch, with many bug
+  fixes, including some important security improvements.
+
+* [Prosody 0.10.1](http://blog.prosody.im/prosody-0-10-1-released/?utm_source=xmppnewsletter):
+  The latest release to the stable branch, containing various bug fixes and improvements.
+
+* [OpenFire 4.2.3](https://discourse.igniterealtime.org/t/openfire-4-2-3-release/8112?utm_source=xmppnewsletter")
+  This release fixes a regression which made it impossible to update plugins from the admin console.
+
+* [Metronome 3.8.27](https://metronome.im/news/3-8-27-maintenance-release?utm_source=xmppnewsletter)
+
+* [Conversations Push Proxy](https://github.com/iNPUTmice/p2), 
+  An XEP-0357: Push Notifications app server that relays push messages between
+  the user’s server and Googles Firebase Cloud Messaging.
+
+### Clients
+
+* [Escalus 4.0.0](https://www.erlang-solutions.com/blog/escalus-4-0-0-faster-and-more-extensive-xmpp-testing.html?utm_source=xmppnewsletter):
+  originally created as a tool to test XMPP servers, it can also be used as a standalone Erlang application.
+  Changes include a new XML viewer, a new XML parser and stanza pipelining.
+
+* [Monal iOS 3.0.0](https://monal.im/blog/monal-ios-3-released/) and then [3.0.1](https://monal.im/blog/ios-3-0-1-released-how-is-push/?utm_source=xmppnewsletter),
+  have been released. The major release adds support for push notifications, a new UI layout, iPhone X support,
+  multi-user chat improvements, conversation synchronization and more.
```

Comment:
Is that still the case? Do you have something I can link to?

Reply:
https://monal.im/blog/updates-and-gdpr/ This seems to be the most recent. The original announcement was in https://monal.im/blog/gdpr-removing-monal-from-the-eu/

Reply:
I'm kind of reluctant to mention the original announcement because to me it looks like fear-mongering (e.g. saying that XMPP federation is no longer legal in the EU). Also, he doesn't need a dedicated DPO, so there goes much of his argumentation.

Reply:
Also, what does "not available" mean? I don't think he can block people from using the client, so it seems only related to his push service.

Reply:
I think it was retracted from the app store for EU (so no more updates). But I don’t know. **Not** linking the original announcement sounds like a good idea.

Given all that, I’d remove the mention altogether at this point. Mentioning it comes with the choice between making the GDPR look bad/contributing to FUD or making Ahu look bad (current  wording, in my opinion). I don’t think we should do either.

Reply:
hmmm, I now included it... and now I'm reluctant to remove it again haha

I was thinking about it back and forth, but I noticed that it's received a lot of comments (looks like it was trending on Hacker News). Given the attention it received I now tend towards including it. Also, people mention some of the misconceptions in the comments.

Reply:
Monal not being available is actually useful information for people who use it or want to use it.

Reply:
Take the "it’s not available" with a grain of salt, since I don’t own an affected device. @ge0rg can you comment on this? I think you have an affected device.

Reply:
It looks like the app is not available on the App Store, but still working through TestFlight for beta users. So I tend to say "not available in the EU".